man/systemd.network.xml

```diff
@@ -1070,19 +1070,19 @@
         <varlistentry>
           <term><varname>InitialCongestionWindow=</varname></term>
           <listitem>
-            <para>The TCP initial congestion window or <literal>InitialCongestionWindow</literal>  is used during the start
-            of a TCP connection. During the start of a TCP session, when a client requests for a resource, the server's initial congestion window
-            determines how many data packets will be sent during the initial burst of data. Takes a number between between 1 and 4294967295 (2^32 - 1).
+            <para>The TCP <literal>InitialCongestionWindow</literal> is used during the start of a TCP connection. During the start
```

Comment:
why write "initial congestion window" as `<literal>InitialCongestionWindow</literal>`?

This "fix" is the wrong way round. We very much prefer the English language over some weird literal terms...